files/default/solr-import-backup.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 
```

Comment:
can you add a comment on what params this script needs as well as a quick description of how it works and a how to for future us.